Java/measureeval/src/test/java/com/lantanagroup/link/measureeval/services/KnowledgeArtifactBuilder.java

```diff
@@ -0,0 +1,283 @@
+                    .setCriteria(new Expression().setLanguage("text/cql-identifier").setExpression("Measure Population Exclusion"));
+            measurePopulationExclusion.setId("measure-population-exclusion");
+            return measurePopulationExclusion;
+        }
+    }
+
+    static class CohortMeasure {
+        private static final String MEASURE_ID = "CohortMeasure";
+        private static final String LIBRARY_ID = "CohortLibrary";
+        private static final String LIBRARY_URL = "https://example.com/Library/" + LIBRARY_ID;
+        public static final String cql = """
+                library CohortLibrary version '1.0.0'
+                
+                using FHIR version '4.0.1'
+                
+                context Patient
+
+                define "Initial Population":
+                  true""";
+        public static Measure measure() {
+            Measure measure = new Measure();
+            measure.addLibrary(LIBRARY_URL);
+            measure.setScoring(new CodeableConcept().addCoding(new Coding().setCode("cohort")));
+            measure.addGroup().addPopulation(MeasurePopulationGroup.initialPopulation());
+            measure.setId(MEASURE_ID);
+            return measure;
+        }
+
+        public static Library library() {
+            Library library = new Library().setVersion("1.0.0").setName(LIBRARY_ID).setUrl(LIBRARY_URL);
+            library.addContent().setContentType("text/cql").setData(cql.getBytes(StandardCharsets.UTF_8));
+            library.setId(LIBRARY_ID);
+            return library;
+        }
+
+        public static Bundle bundle() {
+            Bundle bundle = new Bundle();
+            bundle.addEntry().setResource(library());
+            bundle.addEntry().setResource(measure());
+            return bundle;
+        }
+    }
+
+    static class ProportionMeasure {
+        private static final String MEASURE_ID = "ProportionMeasure";
+        private static final String LIBRARY_ID = "ProportionLibrary";
+        private static final String LIBRARY_URL = "https://example.com/Library/" + LIBRARY_ID;
+        public static final String cql = """
+                library ProportionLibrary version '1.0.0'
+                
+                using FHIR version '4.0.1'
+                
+                context Patient
+
+                define "Initial Population":
+                  true
+                  
+                define "Numerator":
+                  true
+                  
+                define "Numerator Exclusion":
+                  false
+                
+                define "Denominator":
+                  true
+                  
+                define "Denominator Exclusion"
+                  false""";
+        public static Measure measure() {
+            Measure measure = new Measure();
+            measure.addLibrary(LIBRARY_URL);
+            measure.setScoring(new CodeableConcept().addCoding(new Coding().setCode("proportion")));
```

Comment:
We're primarily targeting cohort measures, right now. I like testing additional scoring methods here, but can we make sure that "Cohort" is among the tests, since that's what we're using now?

Reply:
Added additional cohort measure tests